packages/backend/src/libp2p/connectionsManager.ts

```diff
@@ -221,9 +223,14 @@ export class ConnectionsManager extends EventEmitter {
     })
 
     await this.dataServer.listen()
+
     this.io.on('connection', async() => {
-      await this.initTor()
+      if (!this.isTorOnDesktop) {
```

Comment:
I don't like it. As far as I understand, this is always true as the only place you set `isTorOnDesktop` is connectionManager's property default value. Also mind the fact desktop and android behavior will be the same, it's only iOS that's different, so the name of the variable doesn't seem to be accurate